go.mod

```diff
@@ -1,23 +1,90 @@
 module github.com/openstack-k8s-operators/neutron-operator
 
-go 1.13
+go 1.18
 
 require (
-	github.com/blang/semver v3.5.1+incompatible
-	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
+	github.com/go-logr/logr v1.2.3
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.18.1
 	github.com/openshift/api v3.9.0+incompatible
-	github.com/openstack-k8s-operators/cinder-operator v0.0.0-20201015102724-05cd6202cb34 // indirect
-	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20201020115836-d8f73f3ff674
-	github.com/openstack-k8s-operators/lib-common v0.0.0-20201012132655-247b83b2fafa
-	github.com/openstack-k8s-operators/placement-operator v0.0.0-20201016074241-b2b5c6eb0e9c // indirect
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
-	github.com/prometheus/common v0.7.0
-	golang.org/x/net v0.0.0-20200822124328-c89045814202 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v0.18.6
-	sigs.k8s.io/controller-runtime v0.6.2
+	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20220727152756-f13e087e4199
+	github.com/openstack-k8s-operators/keystone-operator/api v0.0.0-20220727152756-f13e087e4199
+	github.com/openstack-k8s-operators/lib-common v0.0.0-20220727151739-0ac9fed768b3
```

Comment:
At the end of last week, we finished reorganizing the module/package layout of `lib-common`.  We'll either want to use that new version here or in a follow-up to this PR.

Reply:
Both option works for me.

Reply:
Okay, one of us can follow-up after we finish with this PR